frame/democracy/README.md

```diff
@@ -1,40 +1,135 @@
 # Democracy Pallet
 
-### Fork
-Our pallet-democracy is a fork of frame's. Please refer to the docs for most of the functionality. Here we describe the main changes made and how that affects interactions and operations with our version.
+- [`democracy::Config`](https://docs.rs/pallet-democracy/latest/pallet_democracy/trait.Config.html)
+- [`Call`](https://docs.rs/pallet-democracy/latest/pallet_democracy/enum.Call.html)
 
-#### Changes
+## Overview
 
-- Proposals are identified by their hash + AssetId.
-- Voting is performed using the proposals AssetId to identify what token to lock.
-- During dispatching, GovernanceRegistry is used to lookup the associated Origin.
-- Different Currency traits are used, such as MutateHold.
-- Orml traits MultiCurrency, MultiLockableCurrency and MultiReservableCurrency are used to support multiple assets.
```

Comment:
please document WHY fork is different from original and please ad hash number into docks you forked from

Reply:
@dzmitry-lahoda  does the comment on line 12 satisfy why it is forked ? "Orml traits MultiCurrency, MultiLockableCurrency and MultiReservableCurrency are used to support multiple assets." or what additions do you suggest ? 

Please explain this "hash number into docks you forked from"

Reply:
Pushed new commit with democracy commit hash and url